libcnb/Cargo.toml

```diff
@@ -16,9 +16,16 @@ workspace = true
 
 [features]
 trace = [
+    "dep:futures-core",
     "dep:opentelemetry",
+    "opentelemetry/trace",
     "dep:opentelemetry_sdk",
-    "dep:opentelemetry-stdout",
+    "opentelemetry_sdk/trace",
+    "dep:opentelemetry-proto",
+    "opentelemetry-proto/trace",
+    "opentelemetry-proto/gen-tonic-messages",
```

Comment:
There's probably not much we can/should do about it, but it seems the `gen-tonic-messages` feature pulls in a whole load of additional deps (tonic, tower, prost etc), eg:
https://github.com/heroku/buildpacks-procfile/pull/262/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eR1078

(I was hoping after https://github.com/open-telemetry/opentelemetry-rust/pull/1569 updating to newer OTel might see a saving in buildpack crates, but the dependency count has increased).

Anyway, we need this functionality (and writing the transformation implementation ourselves would be worse), so there's not much we can do...